conflict_checker.rb

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env ruby
+require_relative 'config/config_reader'
+require_relative 'octokit_client'
+require_relative 'main_controller'
+
+class ConflictChecker
+
+  def initialize
+
+    @repo
+    @repositories = Config_reader.new.get_repos
+    @client = OctokitClient.new
+    @controller = MainController.new
+
+    start
```

Comment:
don't do it in `initialize`. When run should be something like `ConflictChecker.new.start`